src/theme/components/customTable.ts

```diff
@@ -16,10 +16,11 @@ interface BaseStyleProps extends StyleFunctionProps {
 }
 
 const CellBorderColorStyles = () => {
-  const colorValues = useColorModeValue("ui.gray.semi-medium", {
-    base: "ui.gray.xx-dark",
-    md: "dark.ui.border.default",
-  });
+  // Using CSS vars here because of how the values are used in the styles
```

Comment:
Ah, this makes sense. Did we just miss this during dev and qa review?

Reply:
The dark mode styles were not fully QA'ed until Alkim did his final review, when he found that the dark mode colors were totally falling apart. Since only MLN is using dark mode, we pushed the udpates and this is fixing what he was seeing.